samples/word/05-scenarios/content-generation.yaml

```diff
@@ -0,0 +1,507 @@
+    language: typescript
+template:
+    content: |-
+        <div style="height: 1px; width: 1px;">
+            <img src="https://pnptelemetry.azurewebsites.net/pnp-officeaddins/samples/word-add-in-aigc-scriptlab">
+        </div>
+            <div class="wrapper">
+                <div class="main_content">
+
+                    <div id="mainFunc" hidden="hidden">
+                        <div class="back">
+                            <div class="cursor" id="back">
+                                <span>Back</span>
+                            </div>
+                        </div>
+                        <div class="main_func">
+                            <div class="dropdown">
+                                <button class="dropdown_list">Add a title</button>
+                                <div class="dropdown-options">
+                                    <a href="#" id="title-predefined">Add a predefined title</a>
```

Comment:
Let's not include "predefined" content. We have plenty of samples that show that. Let's focus on the AI part.